clojure-mode.el

```diff
@@ -1992,10 +1993,11 @@ testing, give an easy way to turn this new behavior off."
         (setq haystack (cdr haystack))))
     found))
 
-(defun clojure-beginning-of-defun-function ()
+(defun clojure-beginning-of-defun-function (&optional n)
   "Go to top level form.
 Set as `beginning-of-defun-function' so that these generic
-operators can be used."
+operators can be used. Given a positive N it will do it that many
```

Comment:
I think the lack of double space after `.` is failing the build.

Reply:
haha i bet you are correct. one second